### PR TITLE
Clarify and improve setup instructions

### PR DESCRIPTION
**Pull request by frankbu**

### CODE REVIEW

File: content/docs/setup/kubernetes/install/helm/index.md [hunk 1]
```diff
@@ -40,9 +40,18 @@ need to ensure that a CNI plugin is deployed. Refer to [CNI Setup](/docs/setup/k
 for details.
 {{< /tip >}}
 
+## Helm Chart Release Repositories
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/install/helm/index.md]: Only upper case the initial H

File: content/docs/setup/kubernetes/getting-started/index.md [hunk 2]
```diff
@@ -1,14 +1,52 @@
...
+
+1. [Customizable install with Helm](/docs/setup/kubernetes/install/helm/):
+   Choose this option if you want to install Istio for production use, or for performance evaluation.
+
+1. [Istio supported platform](/docs/setup/kubernetes/install/platform/):
+   Choose this option if your Kubernetes platform provides native support for Istio-enabled clusters
+   with a [configuration profile](/docs/setup/kubernetes/additional-setup/config-profiles/)
+   corresponding to your intended use.
+
+After you've chosen an option and created an Istio-enabled cluster, you can then proceed to deploy
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/getting-started/index.md]:
```suggestion
After choosing an option and installing Istio on your cluster, you can deploy
```

File: content/docs/setup/kubernetes/getting-started/index.md [hunk 3]
```diff
@@ -1,14 +1,52 @@
...
+your own application or to experiment with some of our [tasks](/docs/tasks/) and [examples](/docs/examples/).
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/getting-started/index.md]:
```suggestion
your own application or experiment with some of our [tasks](/docs/tasks/) and [examples](/docs/examples/).
```

File: content/docs/setup/kubernetes/getting-started/index.md [hunk 4]
```diff
@@ -1,14 +1,52 @@
...
+
+{{< tip >}}
+If you're running your own application, make sure to
+check the [Requirements for Pods and Services](/docs/setup/kubernetes/additional-setup/requirements/).
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/getting-started/index.md]:
```suggestion
check the [requirements for pods and services](/docs/setup/kubernetes/additional-setup/requirements/).
```

File: content/docs/setup/kubernetes/getting-started/index.md [hunk 5]
```diff
@@ -1,14 +1,52 @@
...
+{{< /tip >}}
+
+When you're ready to consider more advanced Istio setups, check out the following:
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/getting-started/index.md]:
```suggestion
When you're ready to consider more advanced Istio use cases, check out the following resources:
```

File: content/docs/setup/kubernetes/getting-started/index.md [hunk 6]
```diff
@@ -1,14 +1,52 @@
...
+
+- If you want your installation to use Istio's Container Network Interface
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/getting-started/index.md]:
```suggestion
- To install using Istio's Container Network Interface
```

File: content/docs/setup/kubernetes/getting-started/index.md [hunk 7]
```diff
@@ -1,14 +1,52 @@
...
+(CNI) plugin, visit our [CNI guide](/docs/setup/kubernetes/additional-setup/cni/).
+
+- If you want to perform a multicluster setup, visit our
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/getting-started/index.md]:
```suggestion
- To perform a multicluster setup, visit our
```

File: content/docs/setup/kubernetes/getting-started/index.md [hunk 8]
```diff
@@ -1,14 +1,52 @@
...
+[Multicluster installation documents](/docs/setup/kubernetes/install/multicluster/).
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/getting-started/index.md]:
```suggestion
[multicluster installation documents](/docs/setup/kubernetes/install/multicluster/).
```

File: content/docs/setup/kubernetes/getting-started/index.md [hunk 9]
```diff
@@ -1,14 +1,52 @@
...
+
+- To expand your existing mesh with additional containers or VMs not running on
+your mesh's Kubernetes cluster, follow our [Mesh Expansion guide](/docs/setup/kubernetes/additional-setup/mesh-expansion/).
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/getting-started/index.md]:
```suggestion
your mesh's Kubernetes cluster, follow our [mesh expansion guide](/docs/setup/kubernetes/additional-setup/mesh-expansion/).
```

File: content/docs/setup/kubernetes/getting-started/index.md [hunk 10]
```diff
@@ -1,14 +1,52 @@
...
+
+- Adding services requires understanding sidecar injection in detail. Visit our
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/getting-started/index.md]:
```suggestion
- To add services requires detailed understanding of sidecar injection. Visit our
```

File: content/docs/setup/kubernetes/getting-started/index.md [hunk 11]
```diff
@@ -1,14 +1,52 @@
...
+[Installing the Sidecar guide](/docs/setup/kubernetes/additional-setup/sidecar-injection/)
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/getting-started/index.md]:
```suggestion
[sidecar injection guide](/docs/setup/kubernetes/additional-setup/sidecar-injection/)
```

File: content/docs/setup/kubernetes/install/_index.md [hunk 12]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Installation Flows
+title: Installation Guides
 linktitle: Install
-description: Choose the flows that best suit your needs and platform.
+description: Choose the instructions that best suit your needs and platform.
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/install/_index.md]:
```suggestion
description: Choose the guide that best suit your needs and platform.
```

File: content/docs/setup/kubernetes/install/helm/index.md [hunk 13]
```diff
@@ -10,9 +10,9 @@ aliases:
 icon: helm
 ---
 
-Follow this flow to install and configure an Istio mesh for in-depth evaluation or production use.
+Follow these instructions to install and configure an Istio mesh for in-depth evaluation or production use.
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/install/helm/index.md]:
```suggestion
Follow this guide to install and configure an Istio mesh for in-depth evaluation or production use.
```

File: content/docs/setup/kubernetes/install/kubernetes/index.md [hunk 14]
```diff
@@ -7,8 +7,8 @@ aliases:
     - /docs/setup/kubernetes/quick-start/
 ---
 
-Follow this flow to quickly evaluate Istio in a Kubernetes cluster on any platform.
-This flow installs Istio's built-in **demo**
+Follow these instructions to quickly evaluate Istio in a Kubernetes cluster on any platform.
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/install/kubernetes/index.md]:
```suggestion
Follow this guide to quickly evaluate Istio in a Kubernetes cluster on any platform.
```

**Pull request merged**